PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-springcloud/src/main/java/com/microsoft/azure/toolkit/intellij/springcloud/properties/SpringCloudAppPropertiesEditor.java

```diff
@@ -61,16 +62,22 @@ public SpringCloudAppPropertiesEditor(@Nonnull Project project, @Nonnull SpringC
 
     @Override
     protected void rerender() {
-        this.setEnabled(false);
-        AzureTaskManager.getInstance().runLater(() -> {
-            this.formConfig.updateForm(this.draft);
-            this.panelInstances.setApp(this.draft);
-            this.lblSubscription.setText(this.draft.getSubscription().getName());
-            this.lblCluster.setText(this.draft.getParent().getName());
-            this.lblApp.setText(this.draft.getName());
-            this.refreshToolbar();
-            this.formConfig.setValue(this.draft);
-            this.setEnabled(true);
+        final AzureTaskManager tm = AzureTaskManager.getInstance();
+        tm.runLater(() -> {
+            this.setEnabled(false);
+            tm.runOnPooledThread(() -> {
+                this.deploymentDraft = this.draft.updateOrCreateActiveDeployment();
```

Comment:
if there is no deployment in current app (e.g. user delete the active deployment), we will create a new one for user?
Shall we just do nothing and show null here?